src/app/financial-chart/high-volume/financial-chart-high-volume.component.html

```diff
@@ -0,0 +1,6 @@
+<igx-financial-chart height="100%" width="100%"
+	[dataSource]="data"
+	chartType="line"
```

Comment:
Why are you using line here?

Reply:
Is this the hourly one? Line is ok for that, but the daily should use candle.

Reply:
Ok this is the hourly one I guess? Ignore me, then.

Reply:
yes, that one is using line series because it has data with 1-hour interval which produces a lot of data points.